blog/guide-tag-scores-metadata.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: Tags, Scores and Metadata, when to use what?
+---
+
+In Literal, you can add Tags, Scores and Metadata to units like Threads, Steps, Runs and Generations. What do they mean and when do you use which type? 
+
+## Tags
+
+[Tags](/concepts/thread#tags) can be assigned to Threads, Steps and Generations. You can label these units to filter data. Tags are shared across units, application-wide. 
+
```

Comment:
Example tags could be related to data management such as "to review" or "reviewed". Tags could also be related to the nature of the question. For a customer support bot, you could have "billing", "return policy", etc..